models/v1/gardens/gardens_models.go

```diff
@@ -0,0 +1,8 @@
+package gardens_models
+
+type Rules struct {
+	ID             string `bson:"_id" json:"_id"`
```

Comment:
@yukari-n-erb Can change in another PR but I found that you can set the type to `primitive.ObjectID` so go will know that it's an ObjectID. Also, you'll need to include `,omitempty` in the `bson:"_id"` part or else it will be a problem when creating a new garden (I went through this just now :D)
So it should look like this `bson:"_id,omitempty" json:"_id"`.